wiki/Game_mode/osu!catch/en.md

```diff
@@ -74,111 +74,13 @@ The placement of in-game cursor does not matter when playing normally. If [Relax
 
 ## Scoring
 
-*Scoring values can be found in [osu!catch scoring system](/wiki/Score/osu!catch).*
+[The osu!catch score](/wiki/Gameplay/Score/osu!catch) is a weighted sum of all hit objects' values. It depends on the following:
 
-Scoring section details all the intricacies of scoring, including mathematical formula.
+- [Base scoring values](/wiki/Gameplay/Judgement/osu!catch) of objects are fixed and do not depend on [judgement](/wiki/Gameplay/Judgement).
```

Comment:
uh

```suggestion
- [Judgement](/wiki/Gameplay/Judgement) only assigns fixed score values to receive for each caught hit object, because accuracy does not apply in osu!catch.
```

Reply:
ok I've slightly altered your suggestion and added it